### ISSUE TITLE
Homepage & Subpage Photo Header Size Issues

### ISSUE THREAD

**Issue by bigcfk** [uwmadisoncals/uw-madison-wp-2015#5]
### Several Issues Here:
1. If set too small, there is weird overlap with the Header Slide dots.
2. If Header Slides are disabled, this is a moot option.
3. Similarly, Subpages only have a Header image with 'Hero Image on Subpages' - This option could be disabled with some logic.
4. There should be some indication of how big it is. Either an updating pixel count, or just a "fuzzy phrase" to indicate how small/large it is.

**Comment by althe3rd** [comment]
I agree that some more conditional logic would be good, it will take some extra coding to get that applied so I would likely lower the priority of these adjustments until after the new CALS site is ready.

**Comment by bigcfk** [comment]
At the very least, #1 should be addressed. There needs to be a minimum height value, otherwise the Dots and Slide Text overlaps on the title.

**Comment by althe3rd** [comment]
That makes sense. The current minimum value was chosen based on how small you could make the bar while still having everything look good as long as you turn off dots and arrows.

Enlarging that min height will help to prevent issues in the event someone doesn't turn those options off.
